README.md

```diff
@@ -255,3 +259,4 @@ Here's some projects that expand on dotenv. Check them out.
 * [lookenv](https://github.com/RodrigoEspinosa/lookenv)
 * [run.env](https://www.npmjs.com/package/run.env)
 * [dotenv-webpack](https://github.com/mrsteele/dotenv-webpack)
+* [Handy-Browser-Env](https://github.com/yTakkar/Handy-Browser-Env)
```

Comment:
Can you explain how your module expands dotenv and encourages storing configuration in the environment separate from code?

Reply:
When I was developing [React-Instagram-Clone-2.0](https://github.com/yTakkar/React-Instagram-Clone-2.0), I used `dotenv`, it worked like a charm on nodejs as I can access env variables very easily. But there was no package which allowed me to access those env variables on the front-side of the app, so I developed `Handy-Browser-Env`.

Reply:
A key distinction between server-side config and client-side config is that nothing sensitive can go into client-side config because you're distributing it to everyone (no matter how uglified it is, it's still being distributed). Config may still change between deploys for client-side apps but you're better off using JavaScript (or the language of your choice) directly to decide on that config, and it can be stored in git (or your vcs of choice) because, again, this configuration isn't sensitive.

I want to make sure this distinction is clear to developers. I see a lot of issues related to using dotenv in the browser and don't completely understand it. Adding a link to this module won't discourage using dotenv for browser-based projects. **If your module helps you, keep using it.** I'm going to request that this link be removed so that there's no added confusion.

Reply:
Sure, I'll remove it if you want :)